node/node.go

```diff
@@ -622,6 +635,7 @@ func (node *Node) handleNewMiningOutput(miningOutCh <-chan mining.Output) {
 }
 
 func (node *Node) handleNewHeaviestTipSet(ctx context.Context, head types.TipSet) {
+	outboxPolicy := core.NewMessageQueuePolicy(node.Outbox, node.ChainReadStore(), outboxMaxAgeRounds)
```

Comment:
I put it here to emphasises that it was not necessary to make it a member of the node. I can pull it out and make a parameter of this function.

Reply:
Actually, I've moved it only as far as the `Start()` method. We can chat about making it a member of `Node` but I think it's a positive step to avoid that as we pull the node apart.